components/search-form.tsx

```diff
@@ -42,6 +43,7 @@ interface SearchFormProps {
   };
   isLoading?: boolean;
   sortSettings?: SortSetting[];
+  useFilterModal?: boolean;
```

Comment:
```suggestion
  withFilterModal?: boolean;
```

Please rename the props, so it's not easily misunderstood as a custom hook.
Because initially I thought you meant for creating a custom hook when you mention it. 😅

Reply:
Okay, updated